### ISSUE TITLE
Reinforcement Fine-Tuning LLMs with GRPO - DeepLearning.AI

### ISSUE THREAD

**Issue by st01cs** [st01cs/moke-bookmarks#153]
https://www.deeplearning.ai/short-courses/reinforcement-fine-tuning-llms-grpo/?utm_campaign=predibaseC2-launch&utm_medium=headband&utm_source=dlai-homepage

**Comment by st01cs** [comment]
Learn the foundations of reinforcement learning and how to use the Group Relative Policy Optimization (GRPO) algorithm to improve reasoning in large language models.

Design effective reward functions, and learn how rewards are converted into advantages to steer models toward high-quality behavior across multiple use cases.

Learn to use LLM as a Judge for subjective tasks, overcome reward hacking with penalty functions, and calculate the loss function in GRPO.
